pkg/globalnet/controllers/global_egressip_controller.go

```diff
@@ -81,6 +82,7 @@ func NewGlobalEgressIPController(config *syncer.ResourceSyncerConfig, pool *ipam
 
 	for i := range list.Items {
 		err = controller.reserveAllocatedIPs(federator, &list.Items[i], func(reservedIPs []string) error {
+			metrics.RecordAllocateGlobalEgressIPs(pool.GetCIDR(), len(reservedIPs))
```

Comment:
This is on startup to reserve previously allocated IPs so it I don't think we should record this as we're not actually allocating new block. @sridhargaddam WDYT?

Reply:
Good point. 

1. If the metrics value is like a time series information on when the globalIPs are allocated then, as you said, we are just reserving the previously allocated globalIP here, so its not like a new allocation.
2. But if it shows the current number of globalIPs allocated, then we can record it (as the pod was just restarted).

 Lets hear from @skitt

Reply:
The metric is maintained per run of the controller: whenever the controller starts, the metric is 0. If there are already-allocated IPs, we need to adjust the metric accordingly, so this seems correct to me.